src/services/textChanges.ts

```diff
@@ -42,6 +42,10 @@ namespace ts.textChanges {
          * include all trivia between the node and the previous token
          */
         IncludeAll,
+        /**
+         * Only delete trivia one the same line as getStart()
+         */
+        StartLineOnly,

```

Comment:
This should maybe be called `Include` vs the existing `IncludeAll`, whatever reads best to others.